plugins/input/pgraster/pgraster_wkb_reader.cpp

```diff
@@ -193,8 +193,9 @@ void read_data_band(mapnik::raster_ptr raster,
 
   float* data = (float*)image.getBytes();
   double val;
-  val = reader(); // nodata value, need to read anyway
-  if ( hasnodata ) raster->set_nodata(val);
+  double nodataval;
+  nodataval = reader(); // nodata value, need to read anyway
+  if ( hasnodata ) raster->set_nodata(nodataval);
```

Comment:
I prefer how it is done some lines bellow, seems more clear.
```c++
  val = reader();
  if ( hasnodata ) {
      raster->set_nodata(val); // this doesn't actually *do* anything, just sets a value on the raster object
      nodataval = val;
  }
```
but anyway, it is functionally equivalent to the original code, isn't it?